apollo-api/src/main/java/com/apollographql/apollo/api/FileUpload.kt

```diff
@@ -0,0 +1,5 @@
+package com.apollographql.apollo.api
+
+import java.io.File
+
+data class FileUpload(@JvmField val mimetype: String, @JvmField val file: File)
```

Comment:
`@JvmField` is used for backward compatibility. Would be preferable to actually remove this but that would make a breaking change and Java clients need to use `getFile()` etc.